src/app/policies/views/PolicyListView.vue

```diff
@@ -15,95 +15,222 @@
+                    </h3>
 
-          <template v-else>
-            <DataSource
-              v-slot="{ data, error }: PolicyCollectionSource"
-              :src="`/meshes/${route.params.mesh}/policy-path/${route.params.policyPath}?page=${route.params.page}&size=${route.params.size}`"
-            >
-              <DataSource
-                v-slot="{ data: meshInsight }: MeshInsightSource"
-                :src="`/mesh-insights/${route.params.mesh}`"
-              >
-                <PolicyList
-                  :page-number="route.params.page"
-                  :page-size="route.params.size"
-                  :mesh="route.params.mesh"
-                  :current-policy-type="policyTypesData.policies.find((policyType) => policyType.path === route.params.policyPath) ?? policyTypesData.policies[0]"
-                  :policy-types="policyTypesData.policies"
-                  :mesh-insight="meshInsight"
-                  :policy-collection="data"
-                  :policy-error="error"
-                  :is-selected-row="(row) => row.name === route.params.policy"
-                  @change="route.update"
-                />
+                    <p>
+                      {{ t(`policies.type.${policyType.name}.description`, undefined, { defaultMessage: t('policies.collection.description') }) }}
+                    </p>
+                  </div>
+                </div>
+              </KCard>
 
-                <RouterView
-                  v-if="route.params.policy"
-                  v-slot="child"
+              <KCard>
+                <DataLoader
+                  v-slot="{ data }: PolicyCollectionSource"
+                  :src="`/meshes/${route.params.mesh}/policy-path/${route.params.policyPath}?page=${route.params.page}&size=${route.params.size}`"
+                  :loader="false"
                 >
-                  <SummaryView
-                    @close="route.replace({
-                      name: 'policy-list-view',
-                      params: {
-                        mesh: route.params.mesh,
-                        policyPath: route.params.policyPath,
-                      },
-                      query: {
-                        page: route.params.page,
-                        size: route.params.size,
-                      },
-                    })"
+                  <AppCollection
+                    class="policy-collection"
+                    data-testid="policy-collection"
+                    :empty-state-message="t('common.emptyState.message', { type: `${policyType.name} policies` })"
+                    :empty-state-cta-to="t('policies.href.docs', { name: policyType.name })"
+                    :empty-state-cta-text="t('common.documentation')"
+                    :headers="[
+                      { label: 'Name', key: 'name' },
+                      ...(can('use zones') && policyType.isTargetRefBased || policyType.name === 'MeshGateway' ? [{ label: 'Zone', key: 'zone' }] : []),
```

Comment:
Without this `can` the links that appear in this column might not resolve, this is the fix that will need to go to 2.6 as far as I understand it. Please shout if there is something I've missed here.